otx/api/entities/metadata.py

```diff
@@ -59,6 +60,27 @@ def __eq__(self, other):
         return self.name == other.name and self.value == other.value and self.float_type == other.float_type
 
 
+class VideoMetadata(IMetadata):
+    """This class represents metadata of video.
+
+    Args:
+        video_id (ID): ID for video.
+        frame_idx (int): Index for frame.
+    """
+
+    def __init__(self, video_id: ID, frame_idx: int):
```

Comment:
`ID` object is used for objects which are supposed to be stored in repositories like Mongo DB. Why do you need it here? 
I think we should have a consultation with the IAI team before expanding the scope of IDs usage.
Also, common pattern of if usage if self.id = ID(). This constructor guarantees an unique ID, otherwise, the db in geti backend could be ruined.

Reply:
If so, I agree with your idea. I didn't realize the link between ID and DB